code/modules/jobs/job_types/geneticist.dm

```diff
@@ -21,6 +21,7 @@
 
 	display_order = JOB_DISPLAY_ORDER_GENETICIST
 	departments = DEPARTMENT_MEDICAL
+	rpg_title = "Genemancer"
```

Comment:
Genemancer sounds a bit lame. how about Genomancer?

Reply:
I like the original more.